### ISSUE TITLE
M> a few minor typos on homepage

### ISSUE THREAD

**Issue by derek7272** [rchain/Members#122]
On https://www.rchain.coop/#home

Evan Jensen - general counsel not "counsil"

Lisa Rice - space between "companies." and "Lisa". Then the line that begins, "In addition to the Board of Directors and Management team, RChain..." should be on a new paragraph.

David Otto - no semicolon after "counsel". I'd rephrase the last sentence. Maybe, "His credentials include bar admissions in Washington and New York; Harvard University, B.A., Government; Harvard-Shrewsbury Fellowship Recipient, 1981; Fordham University School of Law, J.D.; Commentary Editor, Fordham International Law Journal, 1987"

Greg Heuss - no comma here: "Greg, holds a degree"

**Comment by patrick727** [comment]
thanks @derek7272! the only thing I didnt change was the last sentence on Davids profile but will double check if they are ok with that and go back through. 

Great catches man appreciate it

**Comment by lapin7** [comment]
@derek7272 @patrick727 
Although this issue is fixed, please don't forget to set a budget and to set a reward for this issue.
You can do that through the Gsheet [Pub Member Budget Allocation-Spending](https://docs.google.com/spreadsheets/d/1uxuxx8YN17KCIWcH1cUoGuSm2hAnIya2iAc6wxoaq1o/edit#gid=2052218093)

**Comment by patrick727** [comment]
@lapin7 your right thanks!
